### ISSUE TITLE
FEATURE REQUEST: Change TamperMonkey's icon for an actual monkey face

### ISSUE THREAD

**Issue by LazaroOnline** [Tampermonkey/tampermonkey#1815]
The TamperMonkey icon could be improved.
The name suggest it should be a monkey, just like GreaseMonkey had a nice little monkey icon,
otherwise something wouldn't feel aesthetically correct (it always bugged me the square with the dots).

We can use the MIT-licensed free professional icons from [FluentUI-Emojis](https://www.behance.net/gallery/125956251/Microsoft-Emojis) to make a variation of the monkey face that comes in different versions:
- [3d](https://github.com/microsoft/fluentui-emoji/blob/main/assets/Monkey%20face/3D/monkey_face_3d.png)
- [color](https://github.com/microsoft/fluentui-emoji/blob/main/assets/Monkey%20face/Color/monkey_face_color.svg)
- [flat](https://github.com/microsoft/fluentui-emoji/blob/main/assets/Monkey%20face/Flat/monkey_face_flat.svg)
- [high-contrast](https://github.com/microsoft/fluentui-emoji/blob/main/assets/Monkey%20face/High%20Contrast/monkey_face_high_contrast.svg)

Here is an icon proposal for one possible emoji variation to make the eyes bigger to resemble the 2 big dots of the current icon:
![Microsoft-Fluentui-Emoji-3d-Monkey-Face-3d 512-BigEyes](https://github.com/Tampermonkey/tampermonkey/assets/13026853/bc5b5117-01e7-4275-9c18-778df417fa62)

![Microsoft-Fluentui-Emoji-3d-Monkey-Face-3d 512-BigEyes-Flat](https://github.com/Tampermonkey/tampermonkey/assets/13026853/f8e64ab8-cead-418e-afff-e35e54a36a78)

What do you think @derjanb ? I think it is important that you like the icon and it inspires you to work on the project.
I'm also interested to hear what the community thinks, or if they make other proposals.
In case you want to use the icons I can provide some help making image adjustments, or making it SVG or different sizes.

**Comment by jb222** [comment]
https://www.tampermonkey.net/images/ape.svg

**Comment by LazaroOnline** [comment]
> https://www.tampermonkey.net/images/ape.svg

I've seen that monkey appear in the editor just like Clippy used to in Microsoft Office with a text bubble notification like its my coding buddy. Removing the body from the head, it may serve as an icon, so it is a good alternative that resembles the previous icon squared shape. 
Lets add it as an option for @derjanb to decide:
![ape-head](https://github.com/Tampermonkey/tampermonkey/assets/13026853/5b58fad5-be11-446e-8e90-4e1f213d2eb9)

**Comment by jb222** [comment]
None of those icons will hold in lo-res.
Besides, could your design perhaps look a bit more like a [monkey](https://www.google.com/search?sxsrf=AB5stBiIxfP0-WPJGiEE91KEyThPXziJGQ:1689074516266&q=monkey+icon&tbm=isch&sa=X&ved=2ahUKEwjp2an0xIaAAxVBTKQEHbI-C88Q0pQJegQICRAB)?

**Comment by LazaroOnline** [comment]
Here is how it would look like in the resolution used by Google Chrome:
![ChromeIconSet](https://github.com/Tampermonkey/tampermonkey/assets/13026853/8b9bea62-c140-4220-9f6e-af581d1b4c04)

**Comment by jb222** [comment]
Google Chrome Mac

![Screenshot 2023-07-11 at 20 54 09](https://github.com/Tampermonkey/tampermonkey/assets/61352184/d8dd70fa-f2fd-40df-bfd7-9f92266b7219)

**Comment by LazaroOnline** [comment]
Icon size depends on your OS scale setting and your monitor's DPI (dots-per-inch) also called PPI (pixel-per-inch).
Google Chrome and Firefox on Windows 10 with default screen settings (the recommended scale of 125%) seem to give the extension icon a width of 0.5 centimeters (0.2 inches).
For example, if your monitor has 145 dpi, the icon would be 29 pixels wide on a default Windows 10 installation.
The image above of Mac's Chrome seems to be taken from a low DPI screen and/or a small scale OS setting.
Anyway the trend is to have screens with higher and higher DPIs, as old laptops and monitors get replaced by newer ones.

TLDR: the images have simple enough shapes to be used as extension icons, I even tested scaling it down further to match the Mac-Chrome example and it was still recognizable.

**Comment by jb222** [comment]
> Icon size depends on your OS scale setting and your monitor's DPI (dots-per-inch) also called PPI (pixel-per-inch). Google Chrome and Firefox on Windows 10 with default screen settings (the recommended scale of 125%) seem to give the extension icon a width of 0.5 centimeters (0.2 inches). 

> Anyway the trend is to have screens with higher and higher DPIs, as old laptops and monitors get replaced by newer ones.

Haha, my 24" Eizo screen is only 94ppi and my MacBook has 224ppi which looks a bit sharper. But it makes no difference: there's no way to enlarge the icons in Google Chrome tabs or the bookmarks bar (which is fine as I'd rather not waste screen real estate). See the screenshot with large Finder icons in the background (not my preferred work environment). 

![Screenshot 2023-07-12 at 13 30 23](https://github.com/Tampermonkey/tampermonkey/assets/61352184/96404e32-07e9-4a45-8263-89c8e389e9c7)

BTW, I just tested in Safari and it doesn't show any bookmarks icons at all, only very small ones in drop-down menus.

**Comment by derjanb** [comment]
> Here is how it would look like in the resolution used by Google Chrome: ![ChromeIconSet](https://user-images.githubusercontent.com/13026853/252683750-8b9bea62-c140-4220-9f6e-af581d1b4c04.jpg)

Nice work. However I tend to not change the icon. I got bad feedback when I changed the icon years ago and additionally it would "break" any guide that says something like "click the black square with two dots".
